src/Extension/Exception/ExtendingFailedExceptionHandler.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+/*
+ * This file is part of Flarum.
+ *
+ * For detailed copyright and license information, please view the
+ * LICENSE file that was distributed with this source code.
+ */
+
+namespace Flarum\Extension\Exception;
+
+use Flarum\Foundation\ErrorHandling\HandledError;
+
+class ExtendingFailedExceptionHandler
```

Comment:
I am not sure this is necessary; when we enable an extension, it won't actually be applied until the next refresh. At that point, the error will happen as a boot error, so our error handler won't have an opportunity to intercept. And this isn't used by the logging system. I don't think we need to do anything beyond just throwing the extension.